src/script/components/MessagesList/Message/ContentMessage/ContentMessage.tsx

```diff
@@ -147,6 +149,28 @@ const ContentMessageComponent: React.FC<ContentMessageProps> = ({
   const reactionGroupedByUser = groupByReactionUsers(reactions);
   const reactionsTotalCount = Array.from(reactionGroupedByUser).length;
 
+  const isUserRemovingReaction = (reaction: ReactionType): boolean => {
+    const reactions = message.reactions();
+    const userId = selfId.id;
+    const userReactions = reactions[userId] || '';
+
+    if (userReactions) {
+      const reactionsArr = userReactions.split(',');
+      const reactionIndex = reactionsArr.indexOf(reaction);
+      return reactionIndex !== -1;
+    }
+    // first time reacted
+    return false;
```

Comment:
And for better readability I'll go with smth like that:

```suggestion
    // first time reacted
    if (!userReactions) {
      return false
    }
    
    const reactionsArr = userReactions.split(',');
    const reactionIndex = reactionsArr.indexOf(reaction);
    
    return reactionIndex !== -1;
```

Reply:
already updated with includes and early return